ossdbtoolsservice/driver/types/psycopg_driver.py

```diff
@@ -257,6 +257,12 @@ def get_database_size(self, dbname: str):
         Gets the size of a particular database in MB
         """
 
+    def get_error_message(self, error) -> str:
+        """
+        Get the message from DatabaseError instance 
+        """
+        return error.diag.message_primary
```

Comment:
Can error.diag be None or null? We should check before accessing message_primary, else we ll get an error

Reply:
Actually yeah error.diag can be None, but I wasnt sure if I should add a check for it.

Reply:
Error.diag should not be None as long as it's a psycopg2 error, which we can assume here. Message_primary can be None though, but we handle that outside.